content/hardware/08.edu/solution-and-kits/alvik/tutorials/user-manual/user-manual.md

```diff
@@ -240,6 +346,1018 @@ Open **Arduino Lab for MicroPython** and **connect** Alvik. Then:
+
+```python
+from arduino_alvik import ArduinoAlvik
+from time import sleep_ms
+import sys
+
+alvik = ArduinoAlvik()
+alvik.begin()
+
+while True:
+    try:
+        if alvik.get_touch_any():
+            if alvik.get_touch_up():
+                print("UP")
+            if alvik.get_touch_down():
+                print("DOWN")
+            if alvik.get_touch_left():
+                print("LEFT")
+            if alvik.get_touch_right():
+                print("RIGHT")
+            if alvik.get_touch_ok():
+                print("OK")
+            if alvik.get_touch_cancel():
+                print("CANCEL")
+            if alvik.get_touch_center():
+                print("CENTER")
+
+        sleep_ms(100)
+    except KeyboardInterrupt as e:
+        print('over')
+        alvik.stop()
+        sys.exit()
+```
+
+This example provides a simple way to read and respond to button presses on the Arduino Alvik robot.
+Feel free to expand to it by adding customized responses to each different press.
+
+
+### Detecting Obstacles
+
+These sensors provide detailed readouts from multiple positions around the robot, including **left**, **center-left**, **center, center-right**, **right**, **top**, and **bottom**. Unlike single-point sensors, these distance sensors offer comprehensive zone-based information.
+
+![ToF Sensor top-view](assets/image.png)
+
+The distance sensor array is divided into five distinct zones: left, center-left, center, center-right, and right. Each zone measures distances and is capable of detecting objects up to three meters away. It's important to note that while the sensors can detect objects at greater distances, the reliability of the readings may decrease.
+
+**Read distance**
+
+The `get_distance` function returns the distance readouts from the left, center-left, center, center-right, and right sensors.
+
+**Outputs:**
+`L`: Left sensor readout.
+`CL`: Center-left sensor readout.
+`C`: Center sensor readout.
+`CR`: Center-right sensor readout.
+`R`: Right sensor readout.
+
+**Example Usage**
+
+Here is an example of how to use the distance sensors to get distance measurements:
+
+```python
+from arduino_alvik import ArduinoAlvik
+from time import sleep_ms
+import sys
+
+alvik = ArduinoAlvik()
+alvik.begin()
+
+while True:
+    try:
+        L, CL, C, CR, R = alvik.get_distance()
+        T = alvik.get_distance_top()
+        B = alvik.get_distance_bottom()
+        print(f'T: {T} | B: {B} | L: {L} | CL: {CL} | C: {C} | CR: {CR} | R: {R}')
+        sleep_ms(100)
+    except KeyboardInterrupt as e:
+        print('over')
+        alvik.stop()
+        sys.exit()
+```
+
+In this example, the robot uses its distance sensors to measure distances in its environment. The distance readouts from the left, center-left, center, center-right, right, top, and bottom sensors are printed continuously, allowing the robot to detect obstacles and measure distances accurately.
+
+
+### Following a Line

```

Comment:
I would move this section above to the Out of the box experience and explain here how to program the Line follower sensors same as we are doing with motors and buttons.

Reply:
fine